src/LoRa.cpp

```diff
@@ -144,7 +144,11 @@ int LoRaClass::endPacket()
   writeRegister(REG_OP_MODE, MODE_LONG_RANGE_MODE | MODE_TX);
 
   // wait for TX done
-  while((readRegister(REG_IRQ_FLAGS) & IRQ_TX_DONE_MASK) == 0);
+  while((readRegister(REG_IRQ_FLAGS) & IRQ_TX_DONE_MASK) == 0)
+#ifdef ESP8266
```

Comment:
I think we should remove the `#ifdef` and call yield on all architectures.

Reply:
I wasn't sure if this is working on all platform, thats why I set the guard. If you think so, I prefere!